ingest/ingest_pipeline.py

```diff
@@ -49,7 +49,11 @@
 from mtx import Mtx
 from subsample import SubSample
 from loom import Loom
-from validation.validate_metadata import validate_input_metadata, report_issues
```

Comment:
validate_input_metadata will call all functions needed for a valid metadata file (including collect_jsonschema_errors and validate_collected_ontology_data). If you only call the two functions and not validate_input_metadata, then full metadata file validation is not being performed. Are there  issues with the functions recently added to validate_input_metadata (review_metadata_names and confirm_uniform_units) that need discussion?

Reply:
I think this comment was misplaced and may have to deal with the metadata review and not subsampling. @jlchang is that correct?

Reply:
I've addressed this issue in ddb16a0

Reply:
Thanks Eno. I was worried we'd silently switch back to an earlier state and not run validation in ingest_pipeline as intended. This update addresses my concerns.